.github/workflows/ci.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         env:
           YOCTO_TARGET_ARCH: ${{ matrix.arch }}
         run: |
-          source poky/oe-init-build-env
+          TEMPLATECONF=/opt/yocto/meta-protos/conf/templates source poky/oe-init-build-env
```

Comment:
Does it make sense to move `TEMPLATECONF` to `env:` too?

Reply:
absolutely, i tried several approaches but failed on anything as explain here: https://docs.github.com/en/actions/learn-github-actions/workflow-commands-for-github-actions#setting-an-environment-variable

Reply:
> Does it make sense to move `TEMPLATECONF` to `env:` too (readability)?

Diffs are welcome

Reply:
Let's merge this PR, I'll submit a fixup in case it doesn't break.

Reply:
**Update:** Seems setting it through `env:` doesn't work.